test/e2e/suite_test.go

```diff
@@ -86,10 +86,8 @@ var _ = BeforeSuite(func() {
 
 	if os.Getenv("RUN_LOCAL") == "true" {
 		k8sManager, err := ctrl.NewManager(cfg, ctrl.Options{
-			Scheme:    scheme.Scheme,
-			Namespace: testNamespace,
-			Port:      9443,
```

Comment:
why the port is removed here?

Reply:
Port is part of Webhook server now. It can be set with:
```
WebhookServer : webhook.NewServer(webhook.Options{
			Port:    9443})
```
but zookeeper operator does not have any webhooks and do not setup webhook server, so I don't think we need to start setting this up.